test/src/main/java/com/redhat/lightblue/mongo/test/AbstractMongoCRUDTestController.java

```diff
@@ -45,15 +45,18 @@
 
     @BeforeClass
     public static void prepareMongoDatasources() {
+        if (System.getProperty("mongo.backend") == null) {
+            System.setProperty("mongo.backend", "mongo");
```

Comment:
Making this configurable makes no sense.  For mongo, it will always be the string "mongo".